_redirects

```diff
@@ -1,102 +1,77 @@
 # Community
 
-/repo                  https://github.com/graphql-nexus/nexus                                   301!
+/repo                  https://github.com/graphql-nexus/schema                                  301!
 /tweets                https://twitter.com/nexusgql                                             301!
-/contributing          https://github.com/graphql-nexus/nexus/blob/master/CONTRIBUTING.md       301!
-/changelog             https://github.com/graphql-nexus/nexus/releases                          301!
-/changelog/canary      https://github.com/graphql-nexus/nexus/releases/tag/next                 301!
+/contributing          https://github.com/graphql-nexus/schema/blob/develop/CONTRIBUTING.md     301!
+/changelog             https://github.com/graphql-nexus/schema/releases                         301!
+/changelog/canary      https://github.com/graphql-nexus/schema/releases/tag/next                301!
 /roadmap               https://github.com/orgs/graphql-nexus/projects/1                         301!
```

Comment:
TODO

Reply:
What part here is TODO?

Reply:
Line 8 links to an unused project - we will need to figure out if we have a roadmap, and where we put it.

Reply:
Ahh ok, I'll add a code TODO to help us not forget.

Reply:
to be clear though, the project resource technically exists. I mean that its not a 404 page. But yes its a placeholder that basically just says "no roadmap here" :)